.github/CONTRIBUTING.md

```diff
@@ -18,6 +18,8 @@ There are many ways to contribute to Codecademy Docs:
 - Give feedback in the [Google Form](https://docs.google.com/forms/d/e/1FAIpQLSeqwAiV8C2EjXciqTvPFI-ABxkh9iyy7HL-hnr69GrRW99MiA/viewform).
 - Join the [#CodecademyDocs](https://twitter.com/search?q=%23CodecademyDocs&src=typed_query&f=live) discussion on Twitter.
 
+If you're new to Codecademy Docs and contributing for the first time, it is recommended that you visit the [Issues](https://github.com/Codecademy/docs/issues) section and ask to be assigned to an open issue that interests you. Otherwise, feel free to create a new branch in your fork to create a new entry or edit an existing one.
```

Comment:
```suggestion
If you're new to Codecademy Docs and contributing for the first time, it is recommended that you visit the [Issues](https://github.com/Codecademy/docs/issues) section and ask to be assigned to an open issue that interests you. Otherwise, feel free to submit a [PR](https://www.codecademy.com/resources/docs/git/pull-requests) by creating a new [branch](https://www.codecademy.com/resources/docs/general/git) in your fork to create a new entry or edit an existing one.
```

Reply:
If PR is unclear, maybe we can write it out.

Thank you for updating this!

Reply:
Shouldn't the link to "branch" point to `../git/branch` instead of just `../git`?